cli/src/main/java/Main.java

```diff
@@ -236,9 +239,25 @@ public static void allStatistics() {
         StatsProcessor stats = StatsProcessor.create();
         StatsData allStats = stats.getAllStats();
 
-        //Display aggregate statistics
-        System.out.println("\n- All Statistics -");
-        displayStatistics(allStats);
+        if (statisticalDataExists(allStats.getTotalRolls())) {
+            //Display aggregate statistics
+            System.out.println("\n- All Statistics -");
+            displayStatistics(allStats);
+        }
+    }
+
+    /**
+     * Displays an error if statistical data does not exist.
+     * i.e. Nobody has played the game yet.
+     * @param dataLength   Length of statistical data (total rolls or number of past players).
+     * @return             If the data equals zero (is empty).
+     */
+    public static boolean statisticalDataExists(int dataLength) {
```

Comment:
I agree with you, it's simple enough not to need a method. I intially had:

System.out.println("\nError: Not enough player data to display statistics!");

in both playerStats and allStats method and had a guilty feeling about copy/pasting the same error, so I put it in a method. Maybe I'll keep the checks in the code and the error in the method?